docs/source/usage/plugins/particleCalorimeter.rst

```diff
@@ -154,19 +153,29 @@ Analysis Tools
 The first bin of the energy axis of the calorimeter contains all particle energy less than the minimal detectable energy whereas the last bin contains all particle energy greater than the maximal detectable energy.
 The inner bins map to the actual energy range of the calorimeter.
 
-Sample script for plotting the spatial distribution and the energy distribution:
+To easily access the data, you can use our python module located in ``lib/python/picongpu/extra/plugins/data/calorimeter.py``
 
 .. code:: python
 
-   f = h5.File("<path-to-hdf5-file>")
-   calorimeter = np.array(f["/data/<timestep>/calorimeter"])
+import numpy as np
+import matplotlib.pyplot as plt
 
-   # spatial energy distribution
-   # sum up the energy spectrum
-   plt.imshow(np.sum(calorimeter, axis=0))
-   plt.show()
+from calorimeter import particleCalorimeter
 
-   # energy spectrum
-   # sum up all solid angles
-   plt.plot(np.sum(calorimeter, axis=(1,2)))
-   plt.show()
+# setup access to data
+calObj = particleCalorimeter("./simOutput/e_calorimeter/e_calorimeter_all_%T.bp")
+
+# last bin contains overflow 
+selected_energy_bin = -1
+
+plt.title("selected energy: >{:.1f} keV".format(calObj.getEnergy()[selected_energy_bin]), fontsize=18)
+
+plt.pcolormesh(calObj.getYaw(), calObj.getPitch(), calObj.getData(2000)[selected_energy_bin, :, :])
+
+plt.xlabel(calObj.detector_params["axisLabels"][-1] + r" $[^\circ]$", fontsize=18)
+plt.ylabel(calObj.detector_params["axisLabels"][-2] + r" $[^\circ]$", fontsize=18)
+
+cb = plt.colorbar()
+cb.set_label("energy", fontsize=18)
```

Comment:
The unit of the energy axis is still missing.

Reply:
thanks - fixed